runtime/commonMain/src/kotlinx/serialization/internal/PrimitiveArraysSerializers.kt

```diff
@@ -260,8 +286,13 @@ public class FloatArrayBuilder internal constructor(
  * Encode elements one-by-one, as regular list,
  * unless format's Encoder/Decoder have special handling for this serializer.
  */
+@Deprecated(
+    message = "Deprecated in the favour of DoubleArraySerializer() factory",
+    level = DeprecationLevel.ERROR,
```

Comment:
would it make sense to deprecate `xArrayBuilder` with `HIDDEN` as well?

Reply:
Yeah, works for me